packages/cli/src/utils.ts

```diff
@@ -434,3 +435,63 @@ export async function openInEditor(filePath: string): Promise<void> {
     );
   }
 }
+
+export async function updateLintConfig(path: string): Promise<void> {
```

Comment:
move `updateLintConfig` and `createLintConfig` into the `config-lint` file

Reply:
fixed